### DISCUSSION TITLE
The behavior of "]]" and "][" is the opposite of that in vim

### DISCUSSION THREAD

**Discussion by luooofan** [JetBrains/ideavim#486] [Ideas]
I have noticed that:
https://github.com/JetBrains/ideavim/blob/3606f5ea145d212b7fa1dc071561b45b41866bfe/src/main/java/com/maddyhome/idea/vim/package-info.java#L362-L363

And in vim help manual [29.3  Moving through program](https://vimhelp.org/usr_29.txt.html#29.3) (you can also type ex-command `:help usr_29.txt` in vim)
> To move to the start of the outer block use the "[[" command.  Use "][" to find the end.
> Additionally, "[]" moves backward to the end of a function and "]]" moves forward to the start of the next function.

Quite the opposite.

Maybe we should be consistent with vim?

**Comment by AlexPl292** [comment]
Oh, that seems like an issue. Thank you for the report, I've fixed that!
https://github.com/JetBrains/ideavim/commit/4b49bb4737c50dc200d5f1280c6fcd467d68377d

**Reply by luooofan** [reply]
Great! Thank you!
